mkt/constants/iarc_mappings.py

```diff
@@ -191,6 +191,13 @@
     },
 }
 
+# Change {body: {'key': 'val'}} to {'val': 'key'}. Used to deserialize our
```

Comment:
we should probably add specific example of what these data structures look like. i wrote these and even i sometimes get lost.